modules/profiling-ext/README.txt

```diff
@@ -0,0 +1,48 @@
+Apache Ignite Profiling Module
```

Comment:
Please add license header

Reply:
I have clarified the license header.

Reply:
The project's readme-files does not contain a license header. Why we should add it?